include/RAJA/pattern/kernel/Tile.hpp

```diff
@@ -169,13 +169,13 @@ struct IterableTiler {
     }
 
     RAJA_HOST_DEVICE
-    RAJA_INLINE bool operator!=(const IterableTiler &rhs) const
+    RAJA_INLINE bool operator!=(const IterableTiler::iterator &rhs) const
```

Comment:
Should this just be `iterator`?

Reply:
We could do that also. I thought it was clearer with the more qualified name. What is your preference?

Reply:
I removed the unnecessary type name qualification.